ods_ci/tests/Resources/RHOSi.resource

```diff
@@ -97,3 +97,9 @@ Assign Vars According To Product
         Set Suite Variable    ${OPERATOR_DEPLOYMENT_NAME}    opendatahub-operator-controller-manager
         Set Suite Variable    ${OPERATOR_LABEL_SELECTOR}    control-plane=controller-manager
     END
+
+Create Inference Access Token
```

Comment:
why you moved the kw here?

If it's because my comment about protected kw, I meant to say to add the kw name inside the list of protected keywords